Makefile

```diff
@@ -75,33 +73,35 @@ run: generate fmt vet lint
 deploy: manifests
 	# Remove the certmanager certificate if KSERVE_ENABLE_SELF_SIGNED_CA is not false
 	cd config/default && if [ ${KSERVE_ENABLE_SELF_SIGNED_CA} != false ]; then \
-	${KUSTOMIZE} edit remove resource ../certmanager; \
-	else ${KUSTOMIZE} edit add resource ../certmanager; fi;
-	${KUSTOMIZE} build config/default | kubectl apply -f -
+	echo > ../certmanager/certificate.yaml; \
```

Comment:
This empties the certificate.yaml, `kubectl -k` does not complain?

Reply:
Correct. I tried it locally without any issues.